FlightReservation/components/screens/ItemsMyFlights.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import CrossPlatformIcon from 'react-native-cross-platform-icons';
+import MyFlightsStyles from '../assets/styles/MyFlightsStyles';
+
+const ItemsMyFlights = ({item}) => {
+  return (
+    <>
```

Comment:
This empty brackets doesn't seem to have a functionality.

Reply:
That is a fragment in react and you can use for not add extra multiple nodes to the DOM I think they are using that for not use an extra view... 
https://es.reactjs.org/docs/fragments.html